dw-snackbar.js

```diff
@@ -128,169 +145,206 @@ export class DwSnackbar extends layoutMixin(LitElement) {
           opacity: 0;
         }
 
-        .text{
+        .text {
           padding: 14px 16px;
         }
 
         dw-icon-button,
-        dw-button{
+        dw-button {
           margin-right: 8px;
         }
 
-        a{
+        a {
           text-decoration: none;
         }
 
         @keyframes fadeInUp {
           from {
-            transform: translate3d(0,100%,0)
+            transform: translate3d(0, 100%, 0);
           }
 
           to {
-            transform: translate3d(0,0,0);
-            opacity: 1
+            transform: translate3d(0, 0, 0);
+            opacity: 1;
           }
         }
-      `
+      `,
     ];
   }
 
   static get properties() {
     return {
-
       /**
        * Defaull toast configurations
        */
       defaultConfig: { type: Object },
 
-      /**
-       * Position of the toast at where to show taost
-       * 
-       * e.g. {
-       *    horizontal: 'left', //Posible values: 'left', 'center', 'right'. Default: 'left'
-       *    vertical: 'bottom' //Possible values: 'top', 'bottom'. Default: 'bottom'.
-       *  }
-       */
-      position: { type: Object },
-
       /**
        * Map of currently opened toast with its configuration
        * e.g. { toastId: { message: 'toast' } }
        */
       _toastList: { type: Object },
-      
+
       /**
        * Horizontal position of the toast
        * e.g. 'left', 'center', 'right'
+       * Defaull is 'left' for desktop and 'center' for mobile;
        */
-      _horizontalAlign: { type: String, reflect: true },
-      
+      positionHorizontal: {
+        type: String,
+        reflect: true,
+        attribute: "position-horizontal",
+      },
+
       /**
        * Vertically position of the toast
        * e.g. 'top', 'bottom'
        */
-      _verticalAlign: { type: String, reflect: true }
+      positionVertical: {
+        type: String,
+        reflect: true,
+        attribute: "position-vertical",
+      },
+
+      _disableButton: {
```

Comment:
This property isn't properly taken. `dw-snackbar` really represents a collection of snacks. When multiple snacks are visible and this property becomes `true`, action buttons of all snacks becomes disabled. Is this the behaviour we want? If not, plan it differently and apply.